sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/aio/document_producer.py

```diff
@@ -271,3 +271,51 @@ def _validate_orderby_items(self, res1, res2):
             type2 = _OrderByHelper.getTypeStr(elt2)
             if type1 != type2:
                 raise ValueError("Expected {}, but got {}.".format(type1, type2))
+
+class _NonStreamingDocumentProducer(object):
```

Comment:
This is legacy syntax:
```suggestion
class _NonStreamingDocumentProducer:
```